src/Infrastructure/Services/FraudProtectionService.cs

```diff
@@ -133,6 +157,14 @@ public async Task<SignupResponse> PostSignup(SignUp signup, string correlationId
             return await Read<SignupResponse>(response);
         }
 
+        public async Task<AccountProtection.Response> PostSignupAP(AccountProtection.SignUp signup, string correlationId = null)
+        {
+            string endpoint = string.Format(_settings.Endpoints.SignupAP, _settings.AccountProtectionTenantId, signup.Metadata.SignUpId);
```

Comment:
It's odd to me... does KG not proxy/handle the tenant ID for the merchant facing API the same way they do for the UX APIs?

Reply:
as discussed offline, we'll revisit this change later